### DISCUSSION TITLE
Moving forward 👍

### DISCUSSION THREAD

**Discussion by jycouet** [SergeantWeb/sveltekit-plugin-system#6] [General]
I like the initial spin,thx for sharing. 
Are you using it or it's more of a prototype? 

I think it's close to:
https://github.com/sveltejs/kit/discussions/11054

What's your view on it?

**Comment by SergeantWeb** [comment]
Thanks for your feedback. It's more of a proof of concept at the moment. I haven't implemented it in production yet, but I plan to do soon.

However, yes, it looks like https://github.com/sveltejs/kit/discussions/11054, but without keeping the file structure, which would be a nice improvement !
